assets/src/edit-story/app/story/useStoryReducer/reducers/addPage.js

```diff
@@ -53,7 +53,7 @@ function addPage(state, { page, position }) {
   }
 
   const newPage = {
-    backgroundOverlay: OverlayType.NONE,
+    overlay: OverlayType.NONE,
```

Comment:
This doesn't really seem to be in use at all -- `backgroundOverlay` for a page, will create a code quality issue for removing this from everywhere. Don't want to bloat this PR too much, removing `backgroundOverlay` from the page would include some dashboard folder changes, too.